### DISCUSSION TITLE
Jd.id

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1009] [Personal Filter]
Banner di bagian atas, pindah ke link produk juga muncul

<details>
  <summary>Screenshot</summary>
  
  ![Screenshot_20211022-095653108 (1).jpg](https://user-images.githubusercontent.com/19517680/138386277-ca2c80cc-62ae-4ced-8880-4c977c1bf2d9.jpg)
</details>

**Comment by realodix** [comment]
Sorry Gan @3xploiton3, Saya pribadi menambahkannya ke personal filter dan issue Saya pindahkan ke discussions. 

```
jd.id##.top-banner
m.jd.id##.top-deeplink-bar
m.jd.id#$#.top-deeplink-bar + .viewport .top-header { top: 0!important; }
```

Agan bisa menambahkannya ke personal filter Agan atau langsung melaporkannya ke filter yang memang khusus menangani annoyance seperti [AdGuard Annoyances](https://github.com/AdguardTeam/AdguardFilters/tree/master/AnnoyancesFilter/sections) atau [Fanboy’s Annoyance](https://github.com/easylist/easylist/tree/master/fanboy-addon).

Alasannya karena dalam konteks ini self promotions (dengan berbagai bentuknya di berbagai website) dalam survei banyak juga user yang tidak masalah karena self-promotion walaupun cukup mengganggu tapi juga mengandung manfaat untuk mereka. Karena filter annoyace jadi satu dengan filter utama, tidak ada pilihan lain untuk mereka selain menonaktifkan AdBlockID secara keseluruhan.

Annoyance yang Saya tangani yang benar-benar tidak ada manfaatnya di sisi user:
- Mouse limitation (gak bisa klik kanan, gak bisa select text, dll)
- Key limitation (gak bisa print screen)
- Fake button
- Popup & Overlay yang disalahgunakan untuk menampilkan iklan
- Safelink yang disalahgunakan untuk menampilkan iklan

Jika ada hal lain untuk personal filter, Agan bisa diskusikan di halaman [discussions](https://github.com/realodix/AdBlockID/discussions).
